composer.json

```diff
@@ -7,6 +7,7 @@
     }
   },
   "require": {
+    "php": ">=5.6.0",
```

Comment:
Thanks for bringing this forward, I'm sure in the near future we will upgrade the PHP version